theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -1699,3 +1698,130 @@ def test_conv2d_grad_wrt_weights(self):
                                                                                                   )
                         f_new = theano.function([self.x, self.output_grad_wrt], conv_wrt_w_out)
                         utt.assert_allclose(f_new(input_val, out_grad_val), f_old(input_val, filter_val, out_grad_val))
+
+
+class Grouped_conv_noOptim(unittest.TestCase):
+    def setUp(self):
+        self.num_groups = 3
+        self.border_mode = 'valid'
+        self.subsample = (1, 1)
+        self.img_shape = (5, 6, 5, 5)
+        self.kern_shape = (6, 2, 3, 3)
```

Comment:
> Can you make img_shape and kern_shape a list of shapes and test some non-square image and kernel? Then in each test, loop over those shapes

I was trying to test  values for num_groups == num_channels. But I ran into some trouble with verify_grad with this error 'Non-unit value on shape on a broadcastable dimension.'